js/battle.js

```diff
@@ -3219,7 +3219,16 @@ var Battle = (function () {
 			if (kwargs.prepare || kwargs.anim === 'prepare') {
 				this.prepareMove(pokemon, move, target);
 			} else if (!kwargs.notarget) {
-				(kwargs.anim ? Tools.getMove(kwargs.anim) : move).anim(this, [pokemon.sprite, target.sprite]);
+				move = kwargs.anim ? Tools.getMove(kwargs.anim) : move;
+				if (target.side.active.length > 1 && (move.target === 'allAdjacent' || move.target === 'allAdjacentFoes')) {
```

Comment:
Would the correct way to do that be adding this to all the spread moves in `moves.js` on the server?
`onPrepareHit: function () { this.attrLastMove('[spread]'); },`

I have little knowledge of how the server works, but from looking at examples that should make the server send `[spread]` when a spread move is used, right? Or is there another place where it fits better?